app/models/dns-record.server.ts

```diff
@@ -117,7 +117,7 @@ export async function doesDnsRecordExist(
       username,
       type,
       subdomain,
-      value,
+      value: type === 'CNAME' ? undefined : value,
```

Comment:
It took me a minute to understand what you are doing here :)

Please add some comments what you are doing for the next person coming this way

Reply:
Sure. Done!